srcpkgs/textadept/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'textadept'
+pkgname=textadept
+version=10.6
+revision=1
+archs="i686 x86_64"
```

Comment:
I do not see why to restrict to these archs. Cross builds are broken but looks patchable. Note, that even if cross won't work, this should be replaced with nocross to allow to build for aarch on aarch.

Reply:
@Chocimier It fails the build. ([Log](https://travis-ci.org/void-linux/void-packages/builds/598024290?utm_source=github_status&utm_medium=notification))

Is it possible to make it work? I'm adding `nocross` for now.

Reply:
`vsed -i -e 's/CC = /CC ?= /' -e 's/CFLAGS = /CFLAGS ?= /' -e 's/CXX = /CXX ?= /' -e 's/CXXFLAGS = /CXXFLAGS ?= /' Makefile` in pre_configure is enough to build (I do not have hardware to test on)